zeno/util.py

```diff
@@ -238,3 +238,20 @@ def is_notebook() -> bool:
             return False  # Other type (?)
     except (NameError, ImportError):
         return False  # Probably standard Python interpreter
+
+
+def generate_diff_cols(df: pd.DataFrame, diff_cols: List[ZenoColumn]) -> pd.DataFrame:
```

Comment:
Add docstring here describing function. Look at format of other comments including types, arguments, etc.

If you type """ it should autofill the format!